containers/dotnetcore-mssql/.devcontainer/library-scripts/common-debian.sh

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env bash
```

Comment:
You actually don't need this file either since you aren't building `base.Dockerfile`.  You just will want `library-scripts\azcli-debian.sh`